setup.py

```diff
@@ -51,25 +51,25 @@
                                   "psyclone.tests.*"])
 
 NAME = 'PSyclone'
-AUTHOR = ('Rupert Ford <rupert.ford@stfc.ac.uk>, '
+AUTHOR = ('Rupert Ford, '
           'Andrew Porter <andrew.porter@stfc.ac.uk>, '
           'Sergi Siso <sergi.siso@stfc.ac.uk>')
 AUTHOR_EMAIL = 'andrew.porter@stfc.ac.uk'
 URL = 'https://github.com/stfc/psyclone'
 DOWNLOAD_URL = 'https://github.com/stfc/psyclone'
-DESCRIPTION = ('PSyclone - a compiler for Finite Element/Volume/Difference'
-               ' DSLs in Fortran')
+DESCRIPTION = ('PSyclone - a source-to-source Fortran compiler for HPC '
+               'applications')
 LONG_DESCRIPTION = '''\
-PSyclone is a compiler for Fortran-embedded Domain Specific Languages
-targetting Finite Element/Volume/Difference methods in earth-system
-modelling.
+PSyclone is a source-to-source Fortran compiler designed to programmatically
+optimise, parallelise and instrument HPC applications via user-provided
+transformation scripts.
```

Comment:
Also it might be useful to add:
> PSyclone is used by the UK Meteorology Office for the development of their upcoming new weather model LFRic (link?), and for parallelising the Ocean code NEMO.

Let's show off that we do more than just "hello world"

Reply:
Also a good idea :-)

While we're here, would you like to be listed as a maintainer/author Joerg?

Reply:
Sure.

Reply:
Added.